klog.go

```diff
@@ -472,6 +475,10 @@ type loggingT struct {
 	// with the log-dir option.
 	logFile string
 
+	// When logFile is specified, this limiter makes sure the logFile won't exceeds a certain size. When exceeds, the
+	// logFile will be cleaned up. If this value is negative, no size limitation will be applied to logFile.
```

Comment:
A uint64 can't be negative. Did you mean when it's 0?

Reply:
oh, good catch.